intersection_proximity/preprocessing.py

```diff
@@ -0,0 +1,233 @@
+import pandas as pd
+from dbfread import DBF
+from rtree import index
+import geojson
+from math import isclose
+from shapely.ops import linemerge, split, nearest_points
+from shapely.geometry import MultiLineString, LineString, Point, MultiPoint
+import pickle
+import sys
+from .settings import *
+
+multiplier = 1e5 # multiply all floats by this multiplier so we can compare them
+MIN_SIZE = .0006
+MAX_DIST = 1/multiplier  # maximum distance between intersection and street for a street to to be cut
+
+
+def generate_street_edge_name_map(road_network_dump, osm_way_ids, street_edge_name_file):
```

Comment:
most of the functions in this file are missing comments summarizing what the function does.